### PR TITLE
Screenreader fix for IE11 using design.va.gov

### PR DESCRIPTION
**Pull request by jhonny218**
## Description
Current implementation seems to be stable only on Firefox and Chrome. Implementation was changed to use css style display attribute instead. Both implementations worked seamless in Firefox and Chrome locally but needs further testing in IE11.

This time I'm using the design.va.gov guidelines

## Testing done
Testing passes locally. Needs additional IE11 testing.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms-system/src/js/components/FieldTemplate.jsx [hunk 1]
```diff
@@ -47,32 +47,23 @@ export default function FieldTemplate(props) {
...
-        style={{ display: errorStyle }}
-        className="usa-input-error-message"
-        role="alert"
-        id={errorSpanId}
-      >
-        <span className="sr-only">Error</span> {rawErrors[0]}
-      </span>
-    );
+  if (environment.isDev()) {
+    let errorSpanClass = classNames('usa-input-error-message', {
```
Comment (U-DON) [inline src/platform/forms-system/src/js/components/FieldTemplate.jsx]: I think you're missing the definition of `errorSpan` that uses this class.

Comment (jhonnyoddball) [reply src/platform/forms-system/src/js/components/FieldTemplate.jsx]: it's right below
```    errorSpan = (
      <span className={errorSpanClass} role="alert" id={errorSpanId}>
        <span className="sr-only">Error</span> {rawErrors[0]}
      </span>
    );```

File: Jenkinsfile [hunk 2]
```diff
@@ -39,15 +39,16 @@ node('vetsgov-general-purpose') {
               sh "cd /application && npm run security-check"
             }
           }
-        },
-
+        }
+/*
```
Comment (rianfowler) [inline Jenkinsfile]: add this back when we finish testing

**Review by rianfowler — CHANGES_REQUESTED** [review]

File: jenkins/common.groovy [hunk 3]
```diff
@@ -26,7 +26,7 @@ BUILD_TYPE_OVERRIDE = DRUPAL_MAPPING.get(params.cmsEnvBuildOverride, null)
 
 VAGOV_BUILDTYPES = BUILD_TYPE_OVERRIDE ? [BUILD_TYPE_OVERRIDE] : ALL_VAGOV_BUILDTYPES
 
-DEV_BRANCH = 'master'
+DEV_BRANCH = '314-screenreaderv2'
```
Comment (rianfowler) [inline jenkins/common.groovy]: revert this to `master` after testing on dev

**Review by rianfowler — CHANGES_REQUESTED** [review]

**Pull request closed without merging**